examples/configuration-file/config.json

```diff
@@ -0,0 +1,66 @@
+{
+    "name": "config",
+    "tool-version": "1.0",
+    "description": "A simple script to test configuration files",
+    "command-line": "config.sh [CONFIG_FILE]",
+    "schema-version": "0.4",
+    "container-image": {
+	"type": "docker",
+	"image": "boutiques/examples"
+    },
+    "inputs": [{
+	"id": "num_input",
+	"name": "Number input",
+	"type": "Number",
+	"value-key": "[NUM-INPUT]",
+	"optional": false
+    }, {
+	"id": "list_num_input",
+	"name": "List number input",
+	"type": "Number",
+	"list": true,
+	"value-key": "[LIST-NUM-INPUT]",
+	"optional": false
+    }, {
+	"id": "file_input",
+	"name": "File input",
+        "value-key": "[FILE-INPUT]",
+	"type": "File",
+	"optional": false
+    }, {
+	"id": "optional_input",
+	"name": "Optional input",
+	"type": "String",
+	"value-key": "[OPTION]",
+	"optional": true
+    }],
+    "output-files": [{
+	"id": "output_file",
+	"name": "Output file",
+        "value-key": "[OUTPUT-FILE]",
```

Comment:
Doesn't show in my editor or in https://github.com/glatard/boutiques/blob/02928966fc5e215dfd897a669c5608cad6b38438/examples/configuration-file/config.json